app/views/jst/developer_key.handlebars

```diff
@@ -52,9 +52,11 @@
         </div>
     </td>
     <td class='notes'>
-      <div>
-        {{#t}}{{notes}}{{/t}}
-      </div>
+      {{#if notes}}
+        <div>
+            {{#t}}{{notes}}{{/t}}
```

Comment:
This doesn't need to be wrapped in a translate block, because there is no actual content to translate (no label or anything).